dashboards-reports/public/components/report_definitions/delivery/delivery_constants.tsx

```diff
@@ -24,15 +27,18 @@
  * permissions and limitations under the License.
  */
 
-export const EMAIL_RECIPIENT_OPTIONS = [];
+export const testMessageConfirmationMessage = (
+  <EuiText size='s'>
+    <EuiIcon type='check'/> Test message sent to selected channels. If no test message is received, try again or check your channel settings in Notifications.
+  </EuiText>
+);
 
-export const DELIVERY_TYPE_OPTIONS = [
+// Remove after connecting to backend
```

Comment:
I‘ll say add a TODO to those comments, in case others will work on these task, it's easy to find

Reply:
Will do